manifests/server.pp

```diff
@@ -10,6 +10,7 @@
   $use_xinetd = true,
   $address    = '0.0.0.0',
   $motd_file  = 'UNSET',
+  $pid_file   = '/var/run/rsyncd.pid',
```

Comment:
You can consider a data type to make it self documenting. Since it's a new parameter, there's no risk of incompatibility.
```suggestion
  Variant[Enum['UNSET'], Stdlib::Absolutepath] $pid_file = '/var/run/rsyncd.pid',
```

Reply:
Learned sth new. Done :)

Reply:
Oh, I strongly recommend looking into [data types](https://puppet.com/docs/puppet/6.18/lang_data_type.html). They're great!